packages/account-management/src/hooks/use-location-query.tsx

```diff
@@ -0,0 +1,10 @@
+import { useLocation } from 'react-router-dom';
+import { useMemo } from 'react';
+import { parse } from 'query-string';
+
+export function useLocationQuery<T = any>() {
```

Comment:
The return type is missing here. Since it's returning a function `as T` the return type should probably be `T`:

```suggestion
export function useLocationQuery<T = any>(): T {
```